package.json

```diff
@@ -81,6 +91,24 @@
 						"type": "string"
 					},
 					"description": "The paths to be treated as server-sided directories. Overrides rojo if enabled."
+				},
+				"roblox-ts.command.parameters": {
+					"type": "array",
+					"default": ["-w"],
+					"items": {
+						"type": "string"
+					},
+					"description": "The parameters to be passed into the command triggered from the status bar."
+				},
+				"roblox-ts.command.useNpx": {
```

Comment:
Maybe we should just detect a local install and use that if it exists?

Reply:
The reason I originally added useNpx is so that you wouldn't be forced to use npx (say if you used pnpm to manage your packages, pnpm uses pnpx instead), I'm not sure how that could be resolved without adding more configuration options

Reply:
I'm also not sure about the best method to detect a local install - reading node_packages, reading package.json, etc.

Reply:
should be able to detect if `./node_modules/.bin/rbxtsc` exists

Reply:
then you can use that directly instead of npx

Reply:
Sounds good, I'll implement that